components/form/introduction.rst

```diff
@@ -582,7 +582,7 @@ method:
                 ->add('dueDate', 'date')
                 ->getForm();
 
-            $form->handleRequest($request);
+            $form->handleRequest();
```

Comment:
This should be reverted IMO, as this example shows the usage of the form component inside the Symfony framework. The `HttpFoundationRequestHandler` is registered by default.